docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -1,54 +1,113 @@
-# Getting started with a local GDS Supported Platform development cluster
+# Set up a local GDS Supported Platform instance
 
-These instructions tell you how to set up a local GDS Supported Platform (GSP) cluster and host an app on that cluster.
+The GDS Supported Platform (GSP) is a platform to host Government Digital Service (GDS) services. The GSP is based on [Docker](https://docs.docker.com/) and [Kubernetes](https://kubernetes.io/).
 
-This process is resource-intensive and you must set the resource amount used by Docker to the following:
-- 4 CPUs
-- 8 gigabytes of memory
+These instructions explain how to set up a local instance of the GSP and run the example `govuk-prototype-kit` app on that instance. You can use this process to test your app before deploying that app to a production environment on the GSP. This process has several stages. 
 
-## Build a local GSP cluster
+1. Install [prerequisite software](#install-prerequisite-software).
+1. [Create and run](#create-and-run-a-local-gsp-instance) a local GSP instance.
+1. Create a [Docker image of the `govuk-prototype-kit` app](#create-a-docker-image-of-the-govuk-prototype-kit-app).
+1. Create a [Helm chart](#create-a-helm-chart).
+1. [Deploy the app](#deploy-the-app-on-to-the-local-GSP-instance) on to the local GSP instance.
+1. [Connect](#connect-to-the-app) to the app.
+1. [Destroy](#destroy-the-local-instance) the local instance.
 
-1. Install [Homebrew](https://brew.sh/)
+Setting up a local GSP instance should take no more than 2 hours. Contact the GSP team using the [#re-gsp Slack channel](https://gds.slack.com/messages/CDA7YSP0D) if this process takes longer.
 
-1. Clone the GSP repo:
+<%= warning_text('While setting up or running a local GSP instance, do not:<br>- power down your laptop<br>- put your laptop in standby mode<br>- connect or disconnect the Cisco VPN client') %>
+
+
+## Install prerequisite software
+
+1. Install [Homebrew](https://brew.sh/) and run `brew update` to make sure you have the latest version of Homebrew.
+
+
+
+1. Go to the local repository folder and run `brew bundle` to install the [Homebrew Bundler](https://github.com/Homebrew/homebrew-bundle).
```

Comment:
What about something like “to install the required software” here? The `brew bundle` command does install “Homebrew Bundler” by default in order to be able to install the packages, but it’s the listed packages in the Brewfile that are the important bit, and currently this doesn’t explain that. Neither did the previous words, so blame me. :-)

Reply:
we previously had "install the required software", however feedback from tech writer 2i was to be more specific. How about "install the Homebrew Bundler listed software packages"?

Reply:
"... run `brew bundle` to install the packages listed in the [Brewfile(/path/to/Brewfile)"?

Reply:
LGTM

Reply:
Agreed to @issyl0 to link to https://github.com/alphagov/gsp/blob/master/Brewfile